QUDAConfig.cmake.in

```diff
@@ -25,12 +42,36 @@ if( QUDA_QIO )
 endif()
 
 if( QUDA_OPENMP )
-	find_dependency(OpenMP REQUIRED)
+  find_dependency(OpenMP REQUIRED)
 endif()
 
 
+if( QUDA_TARGET_CUDA )
 enable_language(CUDA)
 find_package(CUDAToolkit REQUIRED)
+elseif(QUDA_TARGET_HIP )
+  # HIP Speific CMake
```

Comment:
no string feelings but wondering how we can avoid cluttering this file as more targets get added